docs/technical-design/endpoints.md

```diff
@@ -0,0 +1,213 @@
+# Endpoints
```

Comment:
```suggestion
# How to create a new endpoint
```
Can you also change the title of this file to be a bit clearer its a how to not a comprehensive list of available endpoints.